README.md

```diff
@@ -52,7 +52,7 @@ vars:
 
 ### Additional metrics
 
-This packages aggreagates and joins activity data onto the prospect model. By default, it does so for email and visit events. If you would like to have aggregates for other events in the `visitor_activity` table, you can see them with the `prospect_metrics_activity_types` variable. You need to use value from the `type_name` column:
+By default, this package aggregates and joins activity data onto the prospect model for email and visit events. If you want to have aggregates for other events in the `visitor_activity` table, use `prospect_metrics_activity_types` variable to see them. Use the `type_name` column value:
```

Comment:
```suggestion
By default, this package aggregates and joins activity data onto the prospect model for email and visit events. If you want to have aggregates for other events in the `visitor_activity` table, use `prospect_metrics_activity_types` variable to generate these aggregates. Use the `type_name` column value:
```

Reply:
I wanted to make the edit above, but I wasn't sure if the user is generating aggregates or just viewing aggregates that already existed.

Reply:
generating aggregates is accurate, thanks!